src/Components/Shared/src/WebEventData.cs

```diff
@@ -196,7 +196,7 @@ private WebEventData(int browserRendererId, ulong eventHandlerId, EventFieldInfo
                     return true;
 
                 case "toggle":
-                    eventArgs = Deserialize<EventArgs>(eventArgsJson, jsonSerializerContext.EventArgs);
+                    eventArgs = EventArgs.Empty;
```

Comment:
Why is it safe to return an empty args object here?

Reply:
`EventArgs` does not have any properties on it. I'm closing this PR in favor of https://github.com/dotnet/aspnetcore/pull/34000 which also has the same change.